frontend/public/components/monitoring.tsx

```diff
@@ -1090,6 +1125,47 @@ const QueryBrowserPage = () => {
     data => _.map(data, ({labels, values}) => ({labels, value: _.get(_.last(values), 'y')}))
   ));
 
+  const onQueryBlur = (e, i) => {
+    if (_.get(e, 'relatedTarget.className', '').includes('query-browser__metrics-dropdown')) {
```

Comment:
Better to add a `data-` or `id` attribute to clearly identify the element instead of using classnames?